app/Components/ChatApp.js

```diff
@@ -36,10 +68,12 @@ class ChatApp extends React.Component {
             <MessageList
               messages={this.state.messages}
             />
+            <div style={ {float:"left", clear: "both"} }
```

Comment:
If you can use flexbox instead, it's easier to nest that within other layouts than float/clear.  If `float` is important here then a comment about why would be helpful.